src/ambiguity_test.c

```diff
@@ -1520,7 +1528,7 @@ s8 determine_sats_addition(ambiguity_test_t *amb_test,
 u8 ambiguity_update_sats(ambiguity_test_t *amb_test, const u8 num_sdiffs,
                          const sdiff_t *sdiffs, const sats_management_t *float_sats,
                          const double *float_mean, const double *float_cov_U,
-                         const double *float_cov_D)
+                         const double *float_cov_D, u8 is_bad_measurement)
```

Comment:
Yeah, we ought to clean it up. I made the basic inversion you mentioned and added some docs, but further refactoring is in order later.